### DISCUSSION TITLE
Android 13 lineageos microg

### DISCUSSION THREAD

**Discussion by imlinstress** [NightscoutFoundation/xDrip#2963] [General]
I recently updated my handset pixel 2 to android 13 to realize that it is impossible to install the app. As soon as one presses the install button the phone reboots.

I also tried different app's versions but in vain.

**Comment by Navid200** [comment]
I have no experience with rooting a phone.
Isn't that the consequence of rooting?  You are at the mercy of those who have created the image because it is not easy for the app developers to recreate the problem unless they use the exact image you have used for rooting your phone.

**Reply by imlinstress** [reply]
Actually it is not rooted, it is just android without the glitchy fingers of google.
But yes i see your point. By the way, i was running the same custom rom android 11 & 12 and never experienced something similar in combination with this app.

Hope someone else can test it and shed more light in this issue.
